samples/test/steps/when.ts

```diff
@@ -96,6 +98,11 @@ When(
   }
 );
 
+When(
+  /^she selects the radio option to "(?<option>.+?)"$/,
```

Comment:
you don't need to use regex here, cucumber can extract param in quotes. 
https://github.com/cucumber/cucumber-expressions#parameter-types